docs/wiki/feature/arsenal.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: wiki
+title: ACE Arsenal
+description: Adds an interface that allow player to edit their loadouts.
+group: feature
+category: general
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 12
+  patch: 0
+---
+
+## 1. Overview
+
+### 1.1 Features
+
+ACE Arsenal has a pretty large number of improvements over BI Virtual Arsenal, here is a non exhaustive list:
```

Comment:
blank line

Reply:
blank lines aren't required there, I also checked two spaces, all three solutions look the same when checking the page.